docker/build_scripts/build_utils.sh

```diff
@@ -30,9 +30,9 @@ function do_python_build {
     if [ $(lex_pyver $py_ver) -lt $(lex_pyver 3.3) ]; then
         local unicode_flags="--enable-unicode=ucs4"
     fi
-    LDFLAGS="-Wl,-rpath /opt/$py_ver/lib" ./configure --prefix=/opt/$py_ver --enable-shared $unicode_flags
-    make -j2
-    make install
+    CFLAGS="-Wformat" LDFLAGS="-Wl,-rpath /opt/$py_ver/lib" ./configure --prefix=/opt/$py_ver --enable-shared $unicode_flags > /dev/null
```

Comment:
Not sure what the CFLAGS is for... maybe `CFLAGS="-w"`? ([-w:    Inhibit all warning messages. ](https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html))